test/cluster/test_system_clients_scan_perf.py

```
@@ -0,0 +1,159 @@
+from test.cluster.test_proxy_protocol import do_cql_handshake
+from test.pylib.scylla_cluster_manager import ScyllaClusterManager
+
+logger = logging.getLogger(__name__)
+
+# Kept modest on purpose - see module docstring. Override via env vars for
+# manual before/after measurement runs with more connections/shards.
+NUM_CONNECTIONS = int(os.environ.get("SCAN_PERF_CONNECTIONS", 2000))
+NUM_SHARDS = int(os.environ.get("SCAN_PERF_SHARDS", 4))
+NUM_CONCURRENT_SCANS = int(os.environ.get("SCAN_PERF_CONCURRENCY", 20))
+
+RESULT_FILE = "/tmp/system_clients_scan_perf_result.txt"
```

Ruff (0.16.3)

[error] Probable insecure usage of temporary file or directory: "/tmp/system_clients_scan_perf_result.txt"

(S108)